src/transformer.js

```diff
@@ -179,7 +179,17 @@
 
       return tree.map(function(o){ return o.value; }).join('');
     },
+
     escape: function(s){ return s.replace(/&/g, '&amp;').replace(/</g, '&lt;'); },
+
+    /** Adds line number info to file statistics */
+    addLineInfoToStat: function(statInfo, token){
+      if('line' in token){
+        statInfo['line'] = token['line'];
```

Comment:
Please change to `.line` unless there's a reason for using dynamic properties this way.